migratekvdb/migration.go

```diff
@@ -608,6 +613,8 @@ func (m *Migrator) migrateBucket(ctx context.Context,
 		select {
 		case <-ctx.Done():
 			return ctx.Err()
+		case <-m.interceptor.ShutdownChannel():
```

Comment:
nit: instead on shutdown we could cancel the context of the migration (which is currently a bare backround context).

Reply:
good idea, way cleaner.